frontends/concrete-python/concrete/fhe/mlir/processors/assign_bit_widths.py

```diff
@@ -396,6 +396,10 @@ def min_max(self, node: Node, preds: List[Node]):
         inputs_and_output_share_precision,
     }
 
+    tfhers_to_native = {
+        inputs_and_output_share_precision,
+    }
```

Comment:
What about from native?
Also is this the condition we really want?

Reply:
We basically want to have the input precision (the one of the tfhers integer) to rule it, as tfhers inputs should come from an external computation. We can't just decrease tfhers int bitwidth, so the native one have to follow.

Reply:
Can't we assign it the precision obtained from the inputset? (e.g., maybe all values that will be given to the circuit is smaller than 10 so it's enough to process 5 chunks from TFHE-rs)

Reply:
It really depends on the usecase, but as I'm currently seeing it, we don't control the tfhers computation part, and we aren't asked to look at it, and what are its outputs, all we know is it's output bitwidth. So we shouldn't have a meaningful inputset to use here to try to know how many chunks should we use.
Cc @BourgerieQuentin wdyt?

Reply:
But that'll still be how the circuit is compiled. If they want to have the full range, they should give it in the inputset.

Reply:
yes it was what we said, so the output precision can differ